_docs/resources/engaging-helix.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Engaging Helix
```

Comment:
To me, this title is ambiguous - is it about engaging with helix assets (design language and code), or engaging with helix team members? Seems like the latter. Maybe consider titling this "Engaging the Helix Team" or something along those lines?

Reply:
good point!

Reply:
+1